src/gui/trackerlogindialog.cpp

```diff
@@ -44,8 +44,6 @@ TrackerLoginDialog::TrackerLoginDialog(QWidget *parent, BitTorrent::TorrentHandl
 
     buttonBox->button(QDialogButtonBox::Ok)->setText(tr("Log in"));
 
-    labelLoginLogo->setPixmap(QPixmap(QLatin1String(":/icons/qbt-theme/encrypted.png")));
```

Comment:
So no icon for this label?

Reply:
I saw it was called "oxygen/encrypted.png" in the past, I updated it to "document-encrypt". Also I borrowed `QPixmap scaledPixmap(const QIcon &icon, const QWidget *widget, const int height);` from #9295, hope it is OK with that.